core/src/main/scala/org/bitcoins/core/util/FileUtil.scala

```diff
@@ -0,0 +1,8 @@
+package org.bitcoins.core.util
+
+object FileUtil {
+
+  def getFileAsSource(fileName: String): scala.io.BufferedSource = {
```

Comment:
Scaladoc? Might be nice to explain what sort of files you can get with this (I assume it's files on the classpath?)